src/crypto/CHIPCryptoPAL.h

```diff
@@ -512,6 +513,54 @@ class AesCcm128Key
     uint8_t bytes[kAES_CCM128_Key_Length];
 };
 
+using AesKeyByteArray = uint8_t[CHIP_CRYPTO_SYMMETRIC_KEY_LENGTH_BYTES];
+
+/**
+ * @brief Platform-specific AES key
+ *
+ * The class represents AES key used by Matter stack (currently, only 128-bit key is supported by
+ * Matter) either in the form of raw key material or key reference, depending on the platform. To
+ * achieve that, it contains an opaque context that can be cast to a specific structure used by
+ * the given platform.
+ */
+class AesKey
```

Comment:
If this is platform specific, it cannot expose *any* storage otherwise it will cost both this storage and the explicit storage.

Reply:
Not sure if I understand, but for platforms that don't use key references this class will be the only place where the key is stored. For keys that do use key references this class will only hold a key reference and the actual key will be stored in the secure environment. This class doesn't make any assumptions about persistent storage of the keys. That is, if we need to store the symmetric keys (which doesn't seem to be the case for session keys) we would need another component to do that.

Reply:
By storage I meant RAM